File: python/divisors.py
def getDivisors(n):
    if n == 1:
        return 1
    divisors = 0
    for i in range(1, n**0.5):
        if n % i == 0:
            divisors += 1
    return divisors * 2

# driver code
print("divisors: " + str(getDivisors(4324320)))

File: pascal/divisors.pas
function getDivisors(n: LongInt): Integer;
var divisors: Integer;
var i: LongInt;
var j: Boolean;
begin
    divisors := 0;
    for i := 1 to n do
    begin
        j := (n mod i = 0);
        if j then divisors := divisors + 1;
    end;
    getDivisors := divisors;
end;

begin
    WriteLn(getDivisors(4324320));
end.